### PR TITLE
standardizing file names

### PR DESCRIPTION
**Pull request by jcasman**
Just fixing files names. Have not added different commands/options/protocols.

### CODE REVIEW

File: fake-theta/examples/frame.jpg
Comment (codetricity) [inline fake-theta/examples/frame.jpg:1]: might be better to delete the test frame

File: real-theta/examples/R0010138.JPG
Comment (codetricity) [inline real-theta/examples/R0010138.JPG:1]: delete test image from repo

**Review by codetricity — CHANGES_REQUESTED** [review]
delete test image and test frame

**Review by codetricity — APPROVED** [review]
populate with more commands in the future.  so far, so good.

**Comment by codetricity** [comment]
@jcasman  Do you see that the issue is still open and assigned to you?
https://github.com/theta360developers/theta-javascript/issues/1#issue-1704439071

For example, top level is this:

top-level

fake-theta
real-theta
x - specific to x, like cameraControlSource
sc2
z1

If you don't agree, then say, "I propose something else"

Please acknowledge that you are still working on the open issue assigned to you.

**Comment by jcasman** [comment]
I'm working on this open issue. I'm intending to include these directories

x
sc2
z1

I am also intending to include more commands from fake-theta.

**Comment by codetricity** [comment]
ok.  Do you also see the other options in the issue?

https://github.com/theta360developers/theta-javascript/issues/1#issue-1704439071

Do you plan to add all of these?

getOptions.js - put a good set of options here
setOffDelay.disable.js
setOffDelay.enable.js - set to default value
sleepDelay.disable.js
sleepDelay.enable.js - set to default value
videoStitching.js
captureMode.image.js
captureMode.video.js

Also, please confirm you are doing this for both fake-theta and real-theta.  Your previous note indicated that you are including the commands only for fake-theta.  What is you plan?

**Comment by jcasman** [comment]
I have already added these options to fake-theta.

getOptions.js - put a good set of options here
setOffDelay.disable.js
setOffDelay.enable.js - set to default value
sleepDelay.disable.js
sleepDelay.enable.js - set to default value
videoStitching.js
captureMode.image.js
captureMode.video.js

I intend to add them to real-theta as well.

And yes, will be adding commands and options to both fake-theta and real-theta.

**Comment by codetricity** [comment]
OK.  please try and complete by Tuesday of next week.  Monday is better because you can assign to the intern and be clearer about the task assignment from the beginning.

**Pull request merged**